docs/docker_compose.md

```diff
@@ -0,0 +1,47 @@
+# Docker Compose
+
+To start Unity Catalog in Docker Compose in one command, install the latest
+version of [Docker Desktop](https://www.docker.com/products/docker-desktop/) and
+run the following from the project's root directory:
+
+```sh
+docker compose up -d
+```
+
+This starts the Unity Catalog server and UI. You can access the UI at
+http://localhost:3000 and the server at http://localhost:8080. Clients like
+DuckDB or Spark running on the host machine will be able to interact on those
+ports with the containers running Unity Catalog.
```

Comment:
[2 / 2]

@dennyglee In the compose specific docs, it explains that both the UI and the backend service are started with `docker compose up`. Do you think this is clear or should we edit the main `README.md` to include a sentence like: 

"Alternatively, the whole app can be started using docker compose. See docker compose docs for details."

Reply:
I like everything here, just would suggest removing the `and the server at http://localhost:8080`.  This is correct but its not needed for a quick start, eh?!  :)